annex/src/example9.cpp

```diff
@@ -0,0 +1,62 @@
+/****************************************************************************
+*	1 event loop
+*	Listener to print the content of the event and the thread id
+*	Publish and subscriber for other types of data: int, my own strct, etc.
+*	Number as the output
+*****************************************************************************/
+
+#include <iostream>
+
+#include <klepsydra/high_performance/event_loop_middleware_provider.h>
+
+#include <thread>
+
+#include <chrono>
+
+# include "int_publisher.h"
+
+# include "position_publisher.h"
+
+int main() {
+	kpsr::high_performance::EventLoopMiddlewareProvider<16> eventloop(nullptr);
+
+	eventloop.start();
+	// main thread id to print here
```

Comment:
Change this comment to what it should do.

Reply:
Done